ui/src/authors/containers/DetailPageContainer.jsx

```diff
@@ -163,14 +170,21 @@ function DetailPage({ record, loading, dispatch, match, publicationsQuery }) {
       <Row className="mb3" type="flex" justify="center">
         <Col xs={24} md={22} lg={21} xxl={18}>
           <ContentBox subTitle="Citation Summary">
-            <Row gutter={{ xs: 0, lg: 32 }}>
-              <Col xs={24} md={24} lg={7}>
-                <CitationsByYearGraphContainer />
-              </Col>
-              <Col xs={24} md={24} lg={17}>
-                <CitationSummaryGraphContainer />
-              </Col>
-            </Row>
+            {publications && publications.size === 0 ? (
```

Comment:
I don't like that we have this thing in some places and we can't use `EmptyOrChildren`, maybe we could change `EmptyOrChildren`  to handle this case? what do you think?

Reply:
We could simply make `EmptyOrChildren` accept a boolean instead of directly an object/collection. We would have to remove the `isEmptyCollection` function (the component would be just like `condition ? content : Empty`) and change all the usages of the component to pass the check that we want to perform.